R/data.R

```diff
@@ -128,11 +128,16 @@
 #' @keywords dataset
 "michigan_school_pairs"
 
-#' @title GV data
+#' @title Cluster-randomized experiment data on voter turnout in cable system markets
 #' 
-#' @description This dataset contains a 10% random sample of individuals from each cable system cluster. 
-#' It specifically includes matched pairs 1–3, which correspond to 7 out of the 85 cable systems. 
-#' The original dataset is derived from a cluster-randomized experiment of Rock the Vote TV advertisements.
+#' @description This dataset is a toy example derived from a cluster-randomized 
+#' field experiment that evaluates the effect of Rock the Vote TV advertisements 
```

Comment:
Suggest that "Rock the Vote TV advertisements" become "\dQuote{Rock the Vote} TV advertisements".